tasks.py

```diff
@@ -160,6 +161,31 @@ def destroy(c, force=False):
         docker_compose(c, 'down --remove-orphans --volumes --rmi=local')
 
 
+@task
+def help(c):
+    """
+    Display some help and available urls for the current project
+    """
+
+    print('Run ' + Fore.GREEN + 'inv help' + Fore.RESET + ' to display this help.')
+    print('Run ' + Fore.GREEN + 'inv -l' + Fore.RESET + ' to list all the available tasks.')
+    print(Fore.GREEN + 'Available URLs for this project:')
+    for domain in [c.root_domain] + c.extra_domains:
+        print(Fore.YELLOW + "* https://" + domain)
+
+    try:
+        response = json.loads(requests.get('http://%s:8080/api/http/routers' % (c.root_domain)).text)
```

Comment:
Can't we use docker API to know which port is exposed here ? I have a project using 8080 for something else so I changed that port.

Reply:
Hello

Actually, the docker-starter is a "template". This means if you modify it (like using a different port for traefik) you will need to modify some integration (like this code).

Getting the port via the docker API should be possible, but this will add some complexity.

I prefer to keep the code as simple as possible. More over, I don't see a valid reason to use traefik on a non-default port, expect dealing with legacy code...